homeassistant/components/switch/__init__.py

```diff
@@ -105,6 +109,16 @@ def today_power_mw(self):
         return None
 
     @property
+    def standby_state(self):
+        """ Is the device on - or in standby. """
+        return None
+
+    @property
+    def sensor_state(self):
+        """ Is the sensor on or off. """
```

Comment:
Cool - will make the changes in the morning.

WeMo Makers have terminals you can wire to monitor a boolean state. It is unrelated to the switch state. Typical example would be to use the switch part of the device to
Operate something and the sensor to monitor. However you can just use the sensor or switch alone.